File: helpers/ts/create_linked_list.ts
namespace ts {
  export class NodeList {
    val: number | null;
    next: NodeList | null;

    constructor(val?: number, next?: NodeList | null) {
      this.val = val === undefined ? null : val;
      this.next = next === undefined ? null : next;
    }
  }
}

function create_linked_list(nums: number[]) {
  const dymmy = new ts.NodeList(0);
  let current = dymmy;

  for (let num of nums) {
    current.next = new ts.NodeList(num);
    current = current.next;
  }

  return dymmy.next
}

console.log(create_linked_list([1,2,3,4,5]));

File: algorithms/helpers/python/create_linked_list.py
class ListNode:
  def __init__(self, val=0, next=None):
    self.val = val
    self.next = next

class LinkedList:
  def create_linked_list(self, nums: list) -> ListNode:
    dummy = ListNode(0)
    current = dummy

    for num in nums:
      current.next = ListNode(num)
      current = current.next
    
    return dummy.next

  def list_linked_list(self, head: ListNode) -> ListNode:
      current = head

      while current:
          print(current.val)
          current = current.next